src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c

```diff
@@ -6,6 +6,8 @@
 
 #define THIS ((EnBomChu*)thisx)
 
+#define SCALE 0.01f
+
```

Comment:
dunno about this define, it would at least require a much larger discussion about how we handle this value in general cause its very common. doesnt seem appropriate to define at the local level

Reply:
i see. I can see its use but still not 100% on board, but can leave for now and see what others think.
It should at least be prefixed to be specific to bombchus, imo. BOMBCHU_SCALE